vendorconsent/tcf2/rangesection.go

```diff
@@ -3,32 +3,40 @@ package vendorconsent
 import (
 	"encoding/binary"
 	"fmt"
-)
 
-func parseRangeSection(data consentMetadata) (*rangeSection, error) {
+	"github.com/prebid/go-gdpr/bitutils"
+)
 
+func parseRangeSection(metadata ConsentMetadata, maxVendorID uint16, startbit uint) (*rangeSection, uint, error) {
+	data := metadata.data
 	// This makes an int from bits 230-241
 	if len(data) < 31 {
-		return nil, fmt.Errorf("vendor consent strings using RangeSections require at least 31 bytes. Got %d", len(data))
+		return nil, 0, fmt.Errorf("vendor consent strings using RangeSections require at least 31 bytes. Got %d", len(data))
+	}
+	numEntries, err := bitutils.ParseUInt12(data, 230)
+	fmt.Printf("numEntries: %d\n", numEntries)
+	fmt.Printf("Old func: %d\n", parseNumEntries(data))
```

Comment:
Are these two `printf`'s intended for debugging?

Reply:
Oh, yes, thanks. I will get rid of them.